scss/v2/components/_promo.scss

```diff
@@ -0,0 +1,107 @@
+.promo {
+    &__container {
+        display: block;
+
+        &:focus {
+            outline: 2px solid $sun-yellow;
+            outline-offset: 2px;
+            text-decoration: none;
+        }
+
+        &:hover {
+            text-decoration: none;
+        }
+    }
+
+    &__background {
+        &--blue-gradient {
+            background-image: linear-gradient(to bottom, $night-blue, $ocean-blue);
+            color: $white;
+        }
+
+        &--plum-gradient {
+            background-image: linear-gradient(to right, $indigo-blue, $plum-purple);
+            color: $white;
+        }
+    }
+
+    &__body {
+        display: flex;
+        height: 100%;
+        align-items: center;
+        justify-content: space-between;
+
+        &:before,
+        &:after {
+          content: '';
+          display: block;
+        }
+
+        &--column {
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: flex-start;
+        }
+    }
+
+    &__copy {
+        @include breakpoint(sm) {
+            flex-basis: 75%;
+        }
+
+        @include breakpoint(md) {
+            flex-basis: 70%;
+        }
+        
+        @include breakpoint(lg) {
+            flex-basis: auto;
+        }
+    }
+
+    &__heading {
+        @include breakpoint(sm) {
+            font-size: 18px;
+            line-height: 1.78;
+        }
+
+        @include breakpoint(md) {
+            font-size: 24px;
+            line-height: 1.78;
+        }
+
+        @include breakpoint(lg) {
+            font-size: 30px;
+            line-height: 1.33;
+            letter-spacing: -0.02px;
+        }
+
+        font-weight: bold;
+        font-stretch: normal;
+        font-style: normal;
+        line-height: 1.33;
+    }
+
+    &__description {
+        width: 100%;
+        word-wrap: break-word;
+        overflow-wrap: break-word;
```

Comment:
Isn't this the same as word-wrap? but CSS3, if so do we need to explicitly declare both?

Just an observation (I agree we should use `break-word` more often and maybe put on all tiles, but we currently only use break-word when links are involved on the website, this will be introducing a new word-wrap type for content. (TBH out of all the tiles the Promo boxes are very wide I am not sure these will actually benefit form this as not likley that we will see a word that is long enough to need breaking)

Reply:
I think that's fair. I think I initially had `break-word` to help with how it looked in IE, but I think one of my other changes has helped that anyway as it looks fine